ts-eks/test/pulumi/index.ts

```diff
@@ -46,8 +45,8 @@ const eksPolicy: StackValidationPolicy = {
 
 
         const k8Deployments = args.resources.filter(r => r.isType(k8s.apps.v1.Deployment));
-        if (k8Deployments.length !== 1) {
-            reportViolation(`Expected 1 kubernetes deployment but found ${k8Deployments.length}`);
+        if (k8Deployments.length !== 3) {
```

Comment:
this will fail upgrade path due to the same thing as in OSS

Reply:
ah, right. So it should be 1 or 3, right?

Reply:
erm actually, do you know offhand what it should be? In OSS it's 1 or 3, but that's with single-exec. So here would it be 3 or 9?